### PR TITLE
Make the repeated messages spam filter give a clearer warning

### PR DESCRIPTION
**Pull request by MrsTonedOne**
https://tgstation13.org/phpBB/viewtopic.php?p=631326

### CODE REVIEW

**Comment by Farquaar** [comment]
Auto-ban? Shouldn't it say auto-mute?

Unless the spam filter straight up bans you now

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -168,11 +168,11 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(CONFIG_GET(flag/automute_on) && !holder && last_message == message)
 		src.last_message_count++
 		if(src.last_message_count >= SPAM_TRIGGER_AUTOMUTE)
-			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. An auto-mute was applied."))
+			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. A mute was automatically applied for the current round. Contact admins to request it's removal."))
```
Comment (jlsnow301) [inline code/modules/client/client_procs.dm:171]:
```suggestion
			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. A mute was automatically applied for the current round. Contact admins to request its removal."))
```

**Comment by MrStonedOne** [comment]
> Auto-ban? Shouldn't it say auto-mute?
> 
> Unless the spam filter straight up bans you now

Intentional.

Users are not entitled to know the exact boundaries or consequences for breaking them, just be warned when they are approaching them.

Auto-ban sends the message that the cold and uncaring gamecode has been designed to quiash you like a bug in short measure should you continue, and it's quite intentional that you aren't told exactly what that entails.

(also: a mute is technically a ban from speaking in IC.)

File: code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -168,11 +168,11 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(CONFIG_GET(flag/automute_on) && !holder && last_message == message)
 		src.last_message_count++
 		if(src.last_message_count >= SPAM_TRIGGER_AUTOMUTE)
-			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. An auto-mute was applied."))
+			to_chat(src, span_danger("You have exceeded the spam filter limit for identical messages. A mute was automatically applied for the current round. Contact admins to request its removal."))
 			cmd_admin_mute(src, mute_type, 1)
 			return TRUE
 		if(src.last_message_count >= SPAM_TRIGGER_WARNING)
-			to_chat(src, span_danger("You are nearing the spam filter limit for identical messages."))
+			to_chat(src, span_danger("You are nearing the auto-ban limit for identical messages."))
```
Comment (cacogen) [inline code/modules/client/client_procs.dm:175]:
```suggestion
			to_chat(src, span_danger("You are nearing the round auto-mute for identical messages."))
```

Comment (MrStonedOne) [reply code/modules/client/client_procs.dm:175]: image

Comment (Mothblocks) [reply code/modules/client/client_procs.dm:175]: you should put that into a code comment honestly

**Pull request merged**